pkg/scheduler/objects/application.go

```diff
@@ -299,7 +314,7 @@ func (sa *Application) timeoutPlaceholderProcessing() {
 			zap.Int("releasing placeholders", len(sa.allocations)),
 			zap.Int("releasing asks", len(sa.requests)))
 		// change the status of the app to Failing. Once all the placeholders are cleaned up, if will be changed to Failed
-		if err := sa.HandleApplicationEvent(FailApplication); err != nil {
+		if err := sa.HandleApplicationEventWithInfo(FailApplication, "InsufficientResources"); err != nil {
```

Comment:
How about changing the message to "ResourceReservationTimeout"?
after all we are doing the resource reservation for the app, this can indicate we were trying but unable to fulfill the requests for a given time.
InsufficientResources on the other hand might be too general
let me know your thought

Reply:
Makes sense to me. Changed to `ResourceReservationTimeout`